server/src/engine/flix.ts

```diff
@@ -60,8 +61,18 @@ export async function start (input: StartEngineInput) {
 
   const { flixFilename, extensionPath, workspaceFiles } = input
 
+  // Get JAVA_HOME enviroment variable
+  const javaHome = process.env.JAVA_HOME
+  if (!javaHome) {
```

Comment:
Instead of erroring completely, perhaps we should just fall back to whatever `java` is in `PATH`.